timesheet_automatic_reminder/models/timesheet_employee_reminder.py

```diff
@@ -0,0 +1,243 @@
+# -*- coding: utf-8 -*-
+# © 2015-2017 Elico corp (www.elico-corp.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import fields, models, api
+
+from common_method import *
+
+
+class TimesheetReport(models.TransientModel):
+    _name = "timesheet.employee.reminder.date"
+    _description = "Timesheet Employee Reminder Date"
+
+    reminder_date = fields.Char(
+        string='Reminder Date',
+    )
+
+    employee_remind_id = fields.Many2one(
+        string='Reminding Employee',
+        comodel_name='timesheet.employee.reminder'
+    )
+
+
+class TimesheetReport(models.TransientModel):
```

Comment:
@dongshukai Class name is not consistent with _name. Think to change one or another.